ngraph/core/reference/include/ngraph/runtime/reference/prelu.hpp

```diff
@@ -24,11 +25,37 @@ namespace ngraph
                        const Shape& arg_shape,
                        const Shape& slope_shape)
             {
+                if (arg_shape.size() >= 2)
+                {
+                    NGRAPH_CHECK(arg_shape[1] == shape_size(slope_shape), "If input shape is >= 2, then second second dim of input needs to be equal to slope shape");
```

Comment:
[Some onnx tests](https://dev.azure.com/openvinoci/dldt/_build/results?buildId=71942&view=logs&j=0609cd82-f9f7-5e70-371c-89fca78031ab&t=d7ecf6fe-a4b9-53ec-d3c3-e8ce906215b2&l=615) fail with this PR changes but [pass ](https://dev.azure.com/openvinoci/dldt/_build/results?buildId=71942&view=logs&j=0609cd82-f9f7-5e70-371c-89fca78031ab&t=d7ecf6fe-a4b9-53ec-d3c3-e8ce906215b2&l=3111)with current master reference and CPU implementation. 
So it is obvious regression.